github/Repository.py

```diff
@@ -2116,6 +2124,18 @@ def get_teams(self):
             None
         )
 
+    def get_topics(self):
+        """
+        :calls: `GET /repos/:owner/:repo/topics <http://developer.github.com/v3/repos>`_
+        :rtype: list of unicode strings
+        """
+        headers, data = self._requester.requestJsonAndCheck(
+            "GET",
+            self.url + "/topics",
+            headers={'Accept': 'application/vnd.github.mercy-preview+json'}
+        )
+        return data
```

Comment:
Maybe `return data['names']` if you want to return a list of strings?
https://developer.github.com/v3/repos/#list-all-topics-for-a-repository

Reply:
agreed on this.  Tried to find other examples where github returns an array of strings as a response and couldn't find one.

Reply:
I am sorry, you are right. Since there is no designated `Topic` class that we can use (neither from Github API docs), I think to be consistent with other calls, we can just output the json.

Reply:
Let's just `return data`